code_generator/.editorconfig

```diff
@@ -10,6 +10,10 @@ charset = utf-8
 insert_final_newline = true
```

Comment:
Any reason this and the config in `tools` are separate? Can we unify everything?

Reply:
I don't think so unless we add soft linking to the setup scripts, which we could do but feels a little brittle. 

Its not ideal to have it replicated though

Reply:
Why can't the single file live in the root of the GDK folder?

Reply:
Oh I misread the docs 🤦‍♂️ We can just have a parent one. Will change